bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/i18n/freeboxos.properties

```diff
@@ -353,6 +359,11 @@ channel-type.freeboxos.wifi-host.label = Access Point
 channel-type.freeboxos.wifi-status.label = Wifi Enabled
 channel-type.freeboxos.wifi-status.description = Indicates whether the wifi network is enabled
 
+# thing types config
+
+thing-type.config.freeboxos.host.mDNS.label = mDNS Name
+thing-type.config.freeboxos.host.mDNS.description = The mDNS name of the network device
+
```

Comment:
To be confirmed but I have the feeling that this config parameter does no more exist ?